app/services/school_data_importer.rb

```diff
@@ -5,16 +5,22 @@
 
 class SchoolDataImporter
   attr_reader :logger
+  attr_reader :start_year
 
-  def initialize(logger)
+  def initialize(logger, start_year = Time.zone.now.year)
     @logger = logger
+    @start_year = start_year
```

Comment:
Do we need both attr reader and a class variable for `start_year`?

Reply:
Pretty sure that's how attr_reader works, right? attr_reader, not attr_accessor

Reply:
Ehh, so let me double check. Do we set a class variable, and then never use it directly again? And instead of using it directly(in a few methods), we access it by using attr_reader? 

Oh, is this how getter and setters work in Rails? Uhh. Fine.

Reply:
You can use an attr_accessor, which creates a getter and setter. But we only want a getter, so I used attr_reader